Assets/script.js

```diff
@@ -3,20 +3,17 @@ var currentWeather = {}
 
 // stores and returns the api key for the weather api
 function getWeatherApiKey() {
-    var weatherApiKey = "20ce152fba104603b4cb45bef144122a";
-    return weatherApiKey;
+    return "20ce152fba104603b4cb45bef144122a";
```

Comment:
change to 
var weatherApiKey = "20ce152fba104603b4cb45bef144122a";
return weatherApiKey;